tests/TestCase.php

```diff
@@ -33,4 +34,21 @@ protected function runCommand(Command $command, array $arguments = [], array $in
 
         return $tester;
     }
+
+    /**
+     * @todo if support for Laravel 5.5 is dropped, this shim can be replaced with actual parent call
+     *
+     * @param string $needle
+     * @param string $haystack
+     * @param string $message
+     */
+    public static function assertStringNotContainsString(string $needle, string $haystack, string $message = ''): void
+    {
+        if (!method_exists(Assert::class, 'assertStringContainsString')) {
+            parent::assertNotContains($needle, $haystack, $message);
```

Comment:
So is the shim the same as the other method? If so, why do we need the shim?

Reply:
Laravel 5.5 requires a version of orchestra/testbench which uses a PHPUnit version which does not feature `assertStringNotContainsString`

Our code calls this one and if the parent is not there, we know it's an older PHPUnit and we call the (now deprecated / old) way of doing it.

Otherwise we just use the actual parent with the code below ⬇️

Reply:
…and the code is gone 😄 

Because we know added `--prefer-lowest` also to the test matrix, we've Laravel 5 versions in our tests without _and with_ the mixin markers, so I employed a different solution not requiring this method/shim which simply skips the test (if `@mixin`s are already present).